### PR TITLE
[adds-url-validation] Adds url validation

### PR DESCRIPTION
**Pull request by erikatakahara**
👍  @luiz
@tisvasconcelos

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -119,6 +120,17 @@ define('form', ['doc'], function($) {
 			}
 		});
 
+		form.find('[type="url"]').each(function(element){
+			var pattern = new RegExp(/^http(s)?:\/\/(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/),
```
Comment (tisvasconcelos) [inline form.js]: What about using a more complete regex, like [this one](http://stackoverflow.com/a/190405) (citing the source, of course)?

@luiz

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "form-amd",
-  "version": "1.0.3-SNAPSHOT",
+  "version": "1.0.3",
```
Comment (tisvasconcelos) [inline bower.json]: Are you going to deploy immediately after merging? If not so, it's better to keep the version unchanged.

@luiz

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "form-amd",
-  "version": "1.0.3-SNAPSHOT",
+  "version": "1.0.3",
```
Comment (tisvasconcelos) [inline package.json]: Are you going to deploy immediately after merging? If not so, it's better to keep the version unchanged.

@luiz

File: form.js (hunk 1 above)
Comment (williammizuta) [inline form.js]: Hmm... so what about using the browser validation by verifying `element.validity.valid`? That way, we keep the same behavior with or without JS.

@luiz

Comment (williammizuta) [inline form.js]: I tested it here on Google Chrome and it works. The fact is: we put the `novalidate` attribute on the form, but we can verify validity at field level (using the expression I mentioned above).

@luiz

**Pull request merged**